ui/components/app/asset-list/asset-list.js

```diff
@@ -181,18 +184,22 @@ const AssetList = ({ onClickAsset }) => {
           });
         }}
       />
-      {balanceIsZero && (
-        <ReceiveTokenLink
-          margin={4}
-          marginBottom={0}
-          marginTop={detectedTokens.length > 0 ? 0 : 4}
-        />
+      {shouldShowTokensLinks && (
+        <>
+          {balanceIsZero && (
+            <ReceiveTokenLink
+              margin={4}
+              marginBottom={0}
+              marginTop={detectedTokens.length > 0 ? 0 : 4}
+            />
+          )}
+          <ImportTokenLink
+            margin={4}
+            marginBottom={2}
+            marginTop={detectedTokens.length > 0 && !balanceIsZero ? 0 : 2}
+          />
+        </>
```

Comment:
We should probably hide this in the non-EVM context for now

Reply:
It's already hidden, since `shouldShowTokensLinks` is set to `false` for `AccountOverviewBtc`